test_validation.py

```diff
@@ -56,3 +56,58 @@ def test_is_valid_integer(self):
         with self.subTest('returns false when input has trailing whitespace'):
             result = self.validator.is_there_whitespace("1 ")
             self.assertFalse(result)
+
+    def test_is_odd(self):
```

Comment:
With the changes made to this PR, are you still able to run the game (I didn't try to pull the PR down myself to check this)? If so, then my suggestion is to create a ticket for get_move and go forward with merging this PR. 
If not, then my suggestion is to make sure the game is still runnable with these changes and then merge it. 
So, my PR approval is conditional on this above question. Let me know if you have questions.

Reply:
It plays! However, it doesn't show a formatted board above a 3x3 size because dynamic board formatting hasn't been implemented yet (that is the scope of the next ticket).